### ISSUE TITLE
Getting error: "server connection dropped: RPC frame LZ4 decompression failure" on a fresh Scylla 4.1 cluster

### ISSUE THREAD

**Issue by gaurarpit12** [scylladb/scylla#7473]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 4.1.2
Cluster size: 2 nodes
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS 7.6

I started up a fresh Scylla cluster comprising of 2 nodes, with scylla 4.1.2 installed on both. I am getting the error **server connection dropped: RPC frame LZ4 decompression failure** on both the nodes. The detailed error is given below:
```Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 24] rpc - client 10.2.7.34:58938: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 16] rpc - client 10.2.7.34:64438: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 11] rpc - client 10.2.7.34:50231: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 15] rpc - client 10.2.7.34:56121: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 5] rpc - client 10.2.7.34:63077: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 14] rpc - client 10.2.7.34:50504: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 22] rpc - client 10.2.7.34:61420: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 24] rpc - client 10.2.7.34:64230: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 26] rpc - client 10.2.7.34:61262: server connection dropped: RPC frame LZ4 decompression failure
Oct 22 06:01:08 eye0707 scylla[10191]:  [shard 6] rpc - client 10.2.7.34:61134: server connection dropped: RPC frame LZ4 decompression failure```

Please let me know how to resolve this

**Comment by tzach** [comment]
Error is similar to https://github.com/scylladb/scylla/issues/6925 which was fixed in 4.1.8

**Comment by slivne** [comment]
This should be fixed in 4.1.8 - please upgrade - if you still have issues please post and we can reopen
